### DISCUSSION TITLE
Show favorites depending on profile

### DISCUSSION THREAD

**Discussion by airald** [devemux86/cruiser#440] [Q&A] [answered]
Since last cruiser improvements regarding favourites I've been collecting a lot of interesting places (for me) in a lot of categories. I think should be useful to hide or show them depending of selected profile. 

I use cruiser in very different situations, and this feature will help a lot for me and I suppose also to other users.

**Comment by devemux86** [comment]
@airald thanks for the suggestion.

Do you mean which favorite groups are visible and from what zoom level?

All this info (and also colors, icons) is now stored inside the favorite files.
To depend on the profiles will require a different implementation and storage.

**Reply by bios4** [reply]
> All this info (and also colors, icons) is now stored inside the favorite files.

Just brainstorming...

You could keep all of this info inside the favorite files, but "activate" them based on a setting!??

E.g. the content of favorite group "biker meeting place" should be shown only in the "Motorcycle" profile, whereas items in "car wash" (just an example) should only be shown when the "Car" profile is active.

By default (and when creating a new one) a favorite group could be "activated" for all available profiles.
A list of checkboxes in each favorite group could define for which profile(s) the group's items should be displayed.

**Reply by devemux86** [reply]
@bios4 thanks, this is an interesting idea.

Like having an extra field in favorites that connects them with the profiles.
And store / export / import this field like all other variables of the favorites.

**Reply by bios4** [reply]
> And store / export / import this field like all other variables of the favorites.

When importing you would need to implement some check whether the defined profile(s) of a favorite group exist on the device where the favorites are imported.

E.g. if I assign a user-defined profile to a group and import these favorites anywhere else where this profile isn't available, it needs to be handled somehow...

**Comment by airald** [comment]
I'm agree with @bios4

**Comment by devemux86** [comment, marked as answer]
This is now implemented and discussed in:
- https://github.com/devemux86/cruiser/discussions/597
